ezidapp/management/commands/proc-search-indexer.py

```diff
@@ -35,6 +43,75 @@ class Command(ezidapp.management.commands.proc_base.AsyncProcessingCommand):
     setting = 'DAEMONS_SEARCH_INDEXER_ENABLED'
     queue = ezidapp.models.async_queue.SearchIndexerQueue
 
+    # note, the method is an overridden version to allow for occasional retries of supposedly "permanent" errors
+    # such as network errors or a service being temporarily down or unresponsive.
+
+    # It is kind of kludgy since it uses an infrequently running check every 5 minutes (300 seconds)
+    # based on (current_time_in_seconds - submit_time_in_seconds) % 300 == 0 for up to 1 day since initial queuing.
+
+    # It's possible it may not hit this exact remainder exactly every 5 minutes since
+    # something may be running and not wake up at exactly 5 minute marks, but it will retry sometime soon since the
```

Comment:
The current time is determined by a few factors:
1. DAEMONS_IDLE_SLEEP=1 sec
2. Batch processing time with batch_size=100
3. DAEMONS_BATCH_SLEEP=1sec
So the chance to trigger the 5 minute marks is very random.